deploy/platform/kubernetes/Makefile.in

```diff
@@ -18,8 +18,8 @@
 
 .EXPORT_ALL_VARIABLES:
 
-NAMESPACE ?= default
+NAMESPACE ?= skywalking-showcase
```

Comment:
> Does the namespace need to be explicitly specified? This should be the default.

@innerpeacez we override the namespace here to `skywalking-showcase`

Reply:
helm install -n `namespace`, which seems to be the default behavior of `-n`

Reply:
I think it is only necessary to define the namespace in the Makefile. There is no need to explicitly specify the namespace in k8s resouces yaml. The namespace will use .Release.Namespace by default

Reply:
OK I got your point now, make sense to me.